lib/routes/sms.js

```diff
@@ -106,8 +106,19 @@ module.exports = (log, isA, error, config, customs, sms) => {
           .then(reply, reply)
 
         function getLocation () {
+          if (request.app.clientAddress === '127.0.0.1') {
```

Comment:
e.g. ON by default, OFF in [.env.dev](https://github.com/mozilla/fxa-auth-server/blob/master/.env.dev)